package-lock.json

```diff
@@ -4828,7 +4823,6 @@
         },
         "node_modules/fsevents/node_modules/balanced-match": {
             "version": "1.0.0",
-            "resolved": "https://registry.npmjs.org/balanced-match/-/balanced-match-1.0.0.tgz",
```

Comment:
What version of npm are you on? I'm on 7.5.3. We shouldn't see a bunch of changes to this file

Reply:
I am on 7.5.5

Reply:
Okay I updated to the latest (7.5.6), checked out this branch, and it will add back all the "resolved" lines when I run "npm install"

@wwlorey @nturinski can y'all try what I tried and see if you see the same behavior?

@neelip you can also try deleting the "node_modules" folder and _then_ running "npm install". Maybe you have something cached causing problems

Reply:
I'm on npm version 7.5.6 and I deleted node_modules before npm installing. I didn't see the resolved lines get added back in but it did remove some integrity lines: [diff.txt](https://github.com/microsoft/vscode-cosmosdb/files/6025503/diff.txt)

Reply:
I am noticing that too. I see all the resolved line added back.